llvm/lib/IR/Function.cpp

```diff
@@ -922,6 +922,15 @@ void Function::setOnlyAccessesInaccessibleMemory() {
   setMemoryEffects(getMemoryEffects() & MemoryEffects::inaccessibleMemOnly());
 }
 
+/// Determine if the call can access only errno memory.
+bool Function::onlyAccessesErrnoMemory() const {
+  return getMemoryEffects().onlyAccessesErrnoMem();
+}
+void Function::setOnlyAccessesErrnoMemory() {
+  setMemoryEffects(getMemoryEffects() &
+                   MemoryEffects::errnoMemOnly(ModRefInfo::Mod));
```

Comment:
I don't think hardcoding this Mod under an API with this name is okay. Please rebase over https://github.com/llvm/llvm-project/commit/a502c656881c8bc28b49a08c29421059d00aca26 and remove these Function APIs, keeping everything local to BuildLibCalls.

Reply:
Definitely, thanks for looking into it.